osgp/platform/osgp-adapter-domain-publiclighting/src/main/resources/osgp-adapter-domain-publiclighting.properties

```diff
@@ -95,7 +95,7 @@ scheduling.task.device.connection.max.allowed.age=2
 # ===   Event Retrieval settings                        ===
 # =========================================================
 # Interval of scheduled task.
-scheduling.task.event.retrieval.cron.expression=0 0/30 0-7,12,16-23 * * *
+scheduling.task.event.retrieval.cron.expression=0 0/30 0-5,14-23 * * *
```

Comment:
This cron expression is wrong. The Quartz job is aware of the timezone, see the property 1 line below. I think it should be  `0 0/30 0-7,16-23 * * *`. Thus removing the trigger at 12 'o clock, and making sure that between 07:00 and 16:00 Dutch local time the mechanics are not bothered by the additional traffic generated by contacting Kaifa devices.

Reply:
This comment is obsolete, Rob Tacx requested another CRON expression: `0 0/30 0-6,17-23 * * *`